spec/datadog/tracing_spec.rb

```diff
@@ -0,0 +1,214 @@
+RSpec.describe Datadog::Tracing do
```

Comment:
Since this module is not supposed to embody behavior -- only expose an API to behavior that is elsewhere, I suggest not testing it in the current manner.

Instead, I suggest only asserting that the other underlying APIs are exercised as expected. This way, we don't need to duplicate their tests.

Reply:
I've rewritten the tests with this suggestion, let me know if it looks better now.

Reply:
Thanks, I like it! :)